akka-actor/src/main/scala-3/akka/util/ccompat/package.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (C) 2018-2021 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.util
+
+/**
+ * INTERNAL API
+ *
+ * Based on https://github.com/scala/scala-collection-compat/blob/master/compat/src/main/scala-2.13/scala/collection/compat/package.scala
+ * but reproduced here so we don't need to add a dependency on this library. It contains much more than we need right now, and is
+ * not promising binary compatibility yet at the time of writing.
+ */
+package object ccompat {
```

Comment:
Shouldn't this be marked as `private[akka]`? I see that it's not in the original either but maybe that is an oversight

Reply:
I didn't know as well!
Turns out that `package object`s are not accepting visibility changes:
```
[error] -- [E088] Syntax Error: /Users/andreatp/workspace/tmp/akka/akka-actor/src/main/scala-3/akka/util/ccompat/package.scala:14:14 
[error] 14 |private[akka] package object ccompat {
[error]    |              ^^^^^^^
[error]    |              Expected start of definition
[error] one error found
```